extensions/cl_intel_mem_force_host_memory.asciidoc

```diff
@@ -46,7 +46,7 @@ Final Draft
 
 == Dependencies
 
-This extension is written against the OpenCL Specification Version 3.0.1-Provisional.
+This extension is written against the OpenCL Specification Version 3.0.1-Experimental.
```

Comment:
FWIW, this version of the specification was released as `3.0.1-Provisional`, so it might be best to leave this one as-is (even though you have to dig pretty deep into OpenCL-Registry git history to even find this version of the spec).

Reply:
I've restored the reference to `3.0.1-Provisional`